api/tokens/tokens.go

```diff
@@ -1,7 +1,17 @@
 package tokens
```

Comment:
The changes in this file are mostly things moving from `internal/command/macaroons.go`.